res/controllers/Denon-MC7000-scripts.js

```diff
@@ -952,35 +959,87 @@ MC7000.censor = function(channel, control, value, status, group) {
         }
     }
 };
-// Param Button for Pitch Play to decrease pitch, or decrease star rating otherwise
-MC7000.StarsDown = function(channel, control, value, status, group) {
-    const deckNumber = script.deckFromGroup(group);
-    const deckIndex = deckNumber - 1;
-    if (value >= 0x00) {
```

Comment:
Small implementation note: This will create a conflict on the next 2.4 -> 2.5 merge, assuming #13588 (a small fix to this logic) is merged. This is expected and can safely be discarded, the new logic already handles button down events correctly.